### PR TITLE
Fix | Publish Asset Files

### PR DESCRIPTION
**Pull request by hamed-aloware**
Changes:

- Publish Asset Files Bug Fixed
- Publish Config File Bug Fixed
- Document Updated

### CODE REVIEW

File: config/fair-queue.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    |--------------------------------------------------------------------------
+    |
+    | These middleware will get attached onto each FairQueue route, giving you
+    | the chance to add your own middleware to this list or change any of
+    | the existing middleware. Or, you can simply stick with this list.
+    |
+    */
+    'middleware' => ['web'],
+
+    'url'      => env('FAIR_QUEUE_REDIS_URL'),
```
[the remark this answers is not in the document]
Comment (hamed-aloware) [reply config/fair-queue.php:16]: You mean `middleware` line?

File: config/fair-queue.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    |--------------------------------------------------------------------------
+    | FairQueue Route Middleware
+    |--------------------------------------------------------------------------
+    |
+    | These middleware will get attached onto each FairQueue route, giving you
+    | the chance to add your own middleware to this list or change any of
+    | the existing middleware. Or, you can simply stick with this list.
+    |
+    */
+    'middleware' => ['web'],
```
Comment (SohrabZ) [inline config/fair-queue.php:14]: should be this:

Screen Shot 2022-03-28 at 7 15 45 AM

File: config/fair-queue.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    |--------------------------------------------------------------------------
+    | FairQueue Route Middleware
+    |--------------------------------------------------------------------------
+    |
+    | These middleware will get attached onto each FairQueue route, giving you
+    | the chance to add your own middleware to this list or change any of
+    | the existing middleware. Or, you can simply stick with this list.
+    |
+    */
+    'middleware' => ['web', 'horizon'],
```
Comment (mohsen-aloware) [inline config/fair-queue.php:14]: still horizon here

File: config/fair-queue.php [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+    'host'     => env('FAIR_QUEUE_REDIS_HOST', '127.0.0.1'),
```
Comment (mohsen-aloware) [inline config/fair-queue.php:16]: oh these parameters should not be here. we need a database config section instead.

File: src/config/fair-queue.php [hunk 5]
```diff
@@ -1,10 +0,0 @@
-<?php
```
Comment (mohsen-aloware) [inline src/config/fair-queue.php:1]: properties inside this file will be merged with the database configuration config file. please don't delete this.

oh BTW what happened to that piece of code inside service provider?!

`$this->mergeConfigFrom(
            __DIR__ . '/config/fair-queue.php',
            'database.redis.fair-queue'
        );`

**Review by mohsen-aloware — CHANGES_REQUESTED** [review]

**Pull request merged**